lib/guest_installation_and_configuration_base.pm

```diff
@@ -1502,13 +1502,16 @@ sub activate_guest_network_bridge_device {
     }
     else {
         if (is_networkmanager) {
-            script_retry("nmcli connection up $args{_bridge_device}", timeout => 60, delay => 15, retry => 3, die => 0);
-            script_retry("nmcli connection up $args{_host_device}", timeout => 60, delay => 15, retry => 3, die => 0);
+            enter_cmd("nmcli connection up $args{_bridge_device}");
+            wait_still_screen(30);
+            enter_cmd("nmcli connection up $args{_host_device}");
+            wait_still_screen(30);
         }
         else {
             script_retry("systemctl restart network", timeout => 60, delay => 15, retry => 3, die => 0);
         }
-        type_string("reset\n");
+        reset_consoles;
+        reset_consoles;
```

Comment:
You really need two ```reset_consoles``` here ?

Reply:
Confirm that current code changes would be fixed up the above test failure - https://openqa.suse.de/tests/18363298#step/unified_guest_installation/255